app/models/academies_api/conversion_project.rb

```diff
@@ -1,9 +1,10 @@
 class AcademiesApi::ConversionProject < AcademiesApi::BaseApiModel
-  attr_accessor :name_of_trust
+  attr_accessor :name_of_trust, :proposed_opening_date
 
   def self.attribute_map
     {
-      name_of_trust: "nameOfTrust"
+      name_of_trust: "nameOfTrust",
+      proposed_opening_date: "proposedAcademyOpeningDate"
```

Comment:
🤔 Wondering whether these models should be responsible for making things Ruby-fied so that we can work with them more easily. I suppose we would have to define a getter and setter like this:

```ruby
class AcademiesApi::ConversionProject < AcademiesApi::BaseApiModel
  attr_accessor :name_of_trust
  attr_writer :proposed_opening_date
  
  def proposed_opening_date
    Date.parse(@proposed_opening_date)
  end

  def self.attribute_map
    {
      name_of_trust: "nameOfTrust",
      proposed_opening_date: "proposedAcademyOpeningDate"
    }
  end
end

```